otcextensions/sdk/auto_scaling/v1/_proxy.py

```diff
@@ -136,6 +139,63 @@ def pause_group(self, group):
         )
         return group.pause(self)
 
+    def wait_for_group(self, group, status='INSERVICE', failures=None,
+                       interval=2, wait=180):
+        """Wait for a group to be in a particular status.
+
+        :param group: The value can be the ID of a group
+            or a :class:`~otcextensions.sdk.auto_scaling.v1.group.Group`
+            instance
+        :param status: Desired status.
+        :param failures:
+            Statuses that would be interpreted as failures.
+        :type failures: :py:class:`list`
+        :param int interval:
+            Number of seconds to wait before to consecutive checks.
+            Default to 2.
+        :param int wait:
+            Maximum number of seconds to wait before the change.
+            Default to 180
+        :return: The resource is returned on success.
+        :raises: :class:`~openstack.exceptions.ResourceTimeout` if transition
+                 to the desired status failed to occur in specified seconds.
+        :raises: :class:`~openstack.exceptions.ResourceFailure` if the resource
+                 has transited to one of the failure statuses.
+        """
+        group = self._get_resource(
```

Comment:
this separate call is not required. Example in https://opendev.org/openstack/openstacksdk/src/branch/master/openstack/compute/v2/_proxy.py#L1100-L1128

Reply:
Done